src/org/opendatakit/briefcase/ui/reused/source/Source.java

```diff
@@ -267,6 +279,23 @@ public String getDescription() {
       return server.getBaseUrl().toString();
     }
 
+    private static void uncheckedBrowse(URL url) {
+      try {
+        getDesktop().browse(url.toURI());
+      } catch (URISyntaxException | IOException e) {
+        throw new BriefcaseException(e);
+      }
+    }
+
+    @Override
+    public void decorate(JLabel label) {
```

Comment:
decorate is getting called twice, so clicking on the link creates two browser tabs.
<img width="651" alt="1" src="https://user-images.githubusercontent.com/55099/49612983-2d2c0580-f95b-11e8-9c9c-398fda441b69.png">
<img width="551" alt="2" src="https://user-images.githubusercontent.com/55099/49612993-33ba7d00-f95b-11e8-8f7e-709fafd8ad4f.png">

Reply:
Good catch, thanks! I'll take a look

Reply:
Solved! Thanks again!